### ISSUE TITLE
is this cli deprecated?

### ISSUE THREAD

**Issue by chrisLovesCode** [feathers-plus/cli#42]
i am wondering if this cli and the whole feathers+ thing is outdated, since its installs an old feathers version and has not received updates for a long time.
are there any issues when upgrading the installed packages?

i'm just starting with feathers and like to choose the right tools :)

**Comment by PatrickLumenus** [comment]
I was just asking myself that question. I too just started using FeathersJS just a few days ago. Feathers+ seems really interesting. Although, I quickly found out its outdated. At this point, I think i'm just going to stick to the regular FeathersJS tools for now.

**Comment by toovy** [comment]
We used this cli with feathers 3 and it was good (the author(s) have put a lot of effort in building it and it has some very good concepts), though we migrated back to the standard cli as it is now in version 4 also capable to generate typescript.

Nevertheless we took over some concepts, but started using them manually (without the generation stuff, e.g. the validation and the JSON schema). My advice so far: better don't use this cli, but as an advanced feathers user have a look at how the cli approaches common feathers problems (just generate a sample project).
